### ISSUE TITLE
package.json is missing

### ISSUE THREAD

**Issue by mahdimp** [fojanb/face-detection#1]
I wasn't able to find package.json in the project

**Comment by fojanb** [comment]
Why do you need package.json?

**Comment by fojanb** [comment]
Check out this line of code :
const faceDetector = new FaceDetector();
This function fetches all face features such as eyes, nose,... in a JSON format. and I stored them in a variable named 'faces'.

**Comment by fojanb** [comment]
I didnt have *package.json in my files but I fetch data via face detection
on my browser(chrome)

**Comment by ditsuke** [comment]
> Check out this line of code :
> const faceDetector = new FaceDetector();
> This function fetches all face features such as eyes, nose,... in a JSON format. and I stored them in a > variable named 'faces'.

Don't you need to import the FaceDetector class from somewhere?

**Comment by fojanb** [comment]
Yes of course. The way you can tell if your browser supports face detection is you can go to the console on any page and type FaceDetector, or typeof FaceDetector. If you get undefined or an error returned, that means your browser does not support it. 
If you are on Chrome, you likely have to go to the flags page in Chrome and turn it on. The way you turn flags on in Chrome is you navigate to chrome://flags.

Then you should search for "Experimental Web Platform features" and enable it.
You will probably have to restart your browser, so go ahead quit it, and restart it, and then you should see typeof FaceDetector work.
From now on you can use FaceDetector() in your js file.
I hope everything is clear now (^_^)

**Comment by ditsuke** [comment]
> 
> 
> Yes of course. The way you can tell if your browser supports face detection is you can go to the console on any page and type FaceDetector, or typeof FaceDetector. If you get undefined or an error returned, that means your browser does not support it.
> If you are on Chrome, you likely have to go to the flags page in Chrome and turn it on. The way you turn flags on in Chrome is you navigate to chrome://flags.
> 
> Then you should search for "Experimental Web Platform features" and enable it.
> You will probably have to restart your browser, so go ahead quit it, and restart it, and then you should see typeof FaceDetector work.
> From now on you can use FaceDetector() in your js file.
> I hope everything is clear now (^_^)

I see now! Thank you, I wasn't aware browsers are moving towards baking shape detection right in. Unfortunately this is not accessible for me since I'm on Firefox which is yet to implement an API experiment or otherwise like Chrome. Just a thought that it might be more accessible to use something like `face-detect.js` in the meanwhile.

**Comment by fojanb** [comment]
Yeah :) you're welcome.

Happy Coding ding ding ding (^_^)
